databricks/sdk/core.py

```diff
@@ -108,6 +109,21 @@ def flatten_dict(d: Dict[str, Any]) -> Dict[str, Any]:
         flattened = dict(flatten_dict(with_fixed_bools))
         return flattened
 
+    def get_oauth_token(self, auth_details: str) -> Token:
```

Comment:
Does it make sense for some of this code to live in `oauth.py`?

Reply:
I did a few changes to reuse some function from `oauth.py`. However, this method still makes sense for 2 reasons:
* Config is not exposed int he cli and we need it
* Even after exposing the config, we can't pass it to `oauth.py` or we create circular dependencies.
So having this here is helpful because it simplifies things. I could move the creation of `params` inside `oauth.py`.